### ISSUE TITLE
add learning checks

### ISSUE THREAD

**Issue by csoderberg** [csoderberg/OSF-Curriculum#4]
Should probably add learning check questions/challenge questions to gauge material acquisition and comprehension

**Comment by evomellor** [comment]
Yeah, I agree. Upon first pass, I think some sort of learning check or self-assessment should be provided. 

I think a good way to go about creating them will be to first look at the learning outcomes and goals, and make sure that each one of them is as specific/measurable as possible. Then the assessments will sort of "create themselves" based on what the participant should be expected to do or know at the end of each module. For example, "Understand the current issues and barrier to reproducibility" could be "Participants should be able to describe (five) common practices in the scientific community that lead to low rates of reproducibility"; and "Understand documentation/organizational issues underpinning this"  could be "Participants should be able to draw the organization of an OSF project and provide examples of how it can address the practices mentioned above."

**Comment by csoderberg** [comment]
Am trying adding a general recap exercise at the end, since we don't really have time in the workshops for detailed learning checks as mentioned above for each section/learning objective. Other modules (such as power) might have more specific learning checks.
